### PR TITLE
Retweet first test tweet poll

### PR DESCRIPTION
**Pull request by stalgiag**
Redoing this because the tweet was accidentally deleted :-P

Related to #4280

### CODE REVIEW

**Comment by gr2m** [comment]
Oh no, something went wrong! It did not include the poll!

https://twitter.com/p5xjs/status/1226592350497050635

I think it might have used an outdated version of the action. Very sorry for that. I think you can delete the tweet again, I'll investigate

**Comment by gr2m** [comment]
For reference 

image

**Comment by stalgiag** [comment]
Yeah that is what I saw the first time as well. I am also getting an email saying that one of the actions failed. I'll grab the link

**Comment by stalgiag** [comment]
I am still getting a `Resource no accessible by integration` [error here](https://github.com/Mattlk13/p5.js/commit/eb895e15ad51b55549170c5d2ff594784770596d/checks?check_suite_id=447076871).

**Comment by gr2m** [comment]
I will setup a test repo to tweet from outside of https://github.com/gr2m/twitter-together/, so my tests won't use the source code.

**Comment by gr2m** [comment]
> I am still getting a `Resource no accessible by integration` [error here](https://github.com/Mattlk13/p5.js/commit/eb895e15ad51b55549170c5d2ff594784770596d/checks?check_suite_id=447076871).

Yeah, that's odd, I thought I resolved that one a while ago.

**Comment by gr2m** [comment]
Very sorry for the trouble, let me do some digging, I'll report back 😭

**Comment by gr2m** [comment]
I should have seen it at https://github.com/processing/p5.js/pull/4280/checks?check_run_id=434869932, the preview does actually not look correct. I'll have another coffee, first ☕️

File: tweets/first-test.tweet [hunk 1]
```diff
@@ -0,0 +1,7 @@
+We are thrilled to be testing a new way to tweet polls directly from Github to expand participation in decision-making. 	
+@gr2m added a polling feature to twitter together https://github.com/gr2m/twitter-together/ to make this possible.	
+
+() This is exciting!	
+() :-D	
+() !!!	
+() ¯\_(ツ)_/¯
```
Comment (gr2m) [inline tweets/first-test.tweet]: Could you please start a new PR, but add spaces between the parentheses? 

```
( ) This is exciting!	
( ) :-D	
( ) !!!	
( ) ¯\_(ツ)_/¯
```

Don't merge it, the review should be enough to check if that was the problem

**Pull request merged**